packages/manager/apps/hub/src/components/hub-order-tracking/HubOrderTracking.component.tsx

```diff
@@ -0,0 +1,220 @@
+import React, {
+  FunctionComponent,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import {
+  OsdsChip,
+  OsdsIcon,
+  OsdsLink,
+  OsdsText,
+  OsdsTile,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_TILE_VARIANT,
+  ODS_CHIP_SIZE,
+  ODS_ICON_NAME,
+  ODS_TEXT_SIZE,
+  ODS_ICON_SIZE,
+} from '@ovhcloud/ods-components';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+} from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import {
+  OdsHTMLAnchorElementRel,
+  OdsHTMLAnchorElementTarget,
+} from '@ovhcloud/ods-common-core';
+import {
+  ButtonType,
+  ShellContext,
+  useOvhTracking,
+} from '@ovh-ux/manager-react-shell-client';
+import style from './style.module.scss';
+import { useFetchLastOrder } from '@/data/hooks/apiOrder/useLastOrder';
+import {
+  ERROR_STATUS,
+  WAITING_PAYMENT_LABEL,
+} from '@/data/api/apiOrder/apiOrder.constants';
+import useDateFormat from '@/hooks/dateFormat/useDateFormat';
+import { Skeletons } from '../skeletons/Skeletons.component';
+import { LastOrderTrackingResponse, OrderHistory } from '@/types/order.type';
+import { TileError } from '../tile-error/TileError.component';
+
+export const HubOrderTracking: FunctionComponent = () => {
+  const { t } = useTranslation('hub/order');
+  const {
+    data: orderDataResponse,
+    isLoading,
+    error,
+    refetch,
+  } = useFetchLastOrder();
+  const context = useContext(ShellContext);
+  const { navigation } = context.shell;
+  const { trackClick } = useOvhTracking();
+
+  const [orderTrackingLink, setOrderTrackingLink] = useState<string>('');
+  const [ordersTrackingLink, setOrdersTrackingLink] = useState<string>('');
+
+  useEffect(() => {
+    (async () => {
+      const urlOrder = (await navigation.getURL(
+        'dedicated',
+        `#/billing/order/${orderDataResponse.orderId}`,
+        {},
+      )) as string;
+
+      const urlOrders = (await navigation.getURL(
+        'dedicated',
+        `#/billing/orders`,
+        {},
+      )) as string;
+
+      setOrderTrackingLink(urlOrder);
+      setOrdersTrackingLink(urlOrders);
+    })();
+  }, [orderDataResponse]);
```

Comment:
I feel like it is clearer to use an <Await> component to handle the link creation / retrieval, as you don't have this superfluous code to handle the fact that `getURL` is asynchronous.

Moreover, using `<Await>` allow you to encapsulate it in a `<Suspense>` and display a Skeleton while URL is being retrieved.

You can find an example here: https://github.com/ovh/manager/pull/12951/files#diff-94fad87b597e8757a7f97eb5939de7aed32d15f34c52ca50a3c1a7f746925982R113